tasks/verify/verify.ts

```diff
@@ -84,10 +84,10 @@ task('verifyAll', 'Verifies all contracts on etherscan')
     }
 
     console.log(`> Verifying all contracts on chain ${chainName}[${chainId}]...`)
-    const { addressBook, graphConfig } = hre.graph({
+    const { addressBook } = hre.graph({
       addressBook: args.addressBook,
-      graphConfig: args.graphConfig,
     })
+    const graphConfig = readConfig(args.graphConfig)
```

Comment:
I had to change the way we load `graphConfig` because the task was not working for me. Let me know if it could be something wrong with my setup.

Reply:
hmm sounds like an issue with the cmd line parameters or your setup, I would revert this change and investigate what the problem was (create an issue for it?). You dont happen to have kept logs no?

Reply:
I've created a ticket for this https://github.com/graphprotocol/contracts/issues/871 thanks